src/Microsoft.OData.Edm/Csdl/CsdlWriterSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="CsdlWriterSettings.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETSTANDARD2_0
+
+namespace Microsoft.OData.Edm.Csdl
+{
+    /// <summary>
+    /// The options/setting for CSDL writer
+    /// </summary>
+    public class CsdlWriterSettings
+    {
+        internal static readonly CsdlWriterSettings Default = new CsdlWriterSettings();

```

Comment:
would it be safe to actually initialize the properties instead of relying on the default 
```suggestion
        internal static readonly CsdlWriterSettings Default = new CsdlWriterSettings { IsIeee754Compatible = false, EnumMemberExpressionAsNumeric = false };

```

Reply:
Emm.... I keep my point. 

---
In reply to: [425971508](https://github.com/OData/odata.net/pull/1775#discussion_r425971508) [](ancestors = 425971508)